aten/src/ATen/native/native_functions.yaml

```diff
@@ -5716,14 +5716,12 @@
 
 - func: geqrf.a(Tensor self, *, Tensor(a!) a, Tensor(b!) tau) -> (Tensor(a!) a, Tensor(b!) tau)
   dispatch:
-    CPU: geqrf_out
-    CUDA: legacy::cuda::_th_geqrf_out
+    CPU, CUDA: geqrf_out
 
 - func: geqrf(Tensor self) -> (Tensor a, Tensor tau)
   variants: method, function
   dispatch:
-    CPU: geqrf
-    CUDA: legacy::cuda::_th_geqrf
+    CPU, CUDA: geqrf
```

Comment:
You don't have to do it this PR, but consider making this structured! https://github.com/pytorch/rfcs/blob/rfc-0005/RFC-0005-structured-kernel-definitions.md